plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -52,9 +57,42 @@ const CatalogPageContents = () => {
   const userId = useApi(identityApiRef).getUserId();
   const [selectedTab, setSelectedTab] = useState<string>();
   const [selectedSidebarItem, setSelectedSidebarItem] = useState<string>();
+  const [entitiesState, setEntitiesState] = useState<Entity[]>([]);
+  const [errorState, setError] = useState<Error | undefined>();
+
+  useEffect(() => {
+    setError(error);
+    setEntitiesState(matchingEntities);
+  }, [error, matchingEntities]);
+  const catalogApi = useApi(catalogApiRef);
   const orgName =
     useApi(configApiRef).getOptionalString('organization.name') ?? 'Company';
 
+  const addMockData = async () => {
+    try {
+      const dummyEntities = [
+        'artist-lookup-component.yaml',
+        'playback-order-component.yaml',
+        'podcast-api-component.yaml',
+        'queue-proxy-component.yaml',
+        'searcher-component.yaml',
+        'playback-lib-component.yaml',
+        'www-artist-component.yaml',
+        'shuffle-api-component.yaml',
+      ];
```

Comment:
Would it be better if these file names were loaded from config? That way they could be overridden with sample metadata which is more useful to a particular industry or company.

Reply:
@dtuite How do I load these files from config? I'll look into it.
I am new to the codebase if you could point me to the folder that would help.

Reply:
This example might help: https://github.com/spotify/backstage/blob/master/packages/app/src/apis.ts#L71

The config is loaded from `app-config.yaml` in the root of the project.